packages/components/src/index.ts

```diff
@@ -2,13 +2,17 @@ export * from "./badge/index.ts";
 export * from "./buttons/index.ts";
 export * from "./checkbox/index.ts";
 export * from "./chip/index.ts";
+export * from "./divider/index.ts";
```

Comment:
Divider is a component so the name of its folder should start with a "D".

Reply:
well we had a conversation about this before and I thought we would have all folder names start with a lowercase letter. Maybe I didn't understand or it was unclear. We should make a decision about this though. @alexasselin008 , any thoughts on this? I just want to make sure everyone is aligned before I make the change.

Reply:
I couldn't find a proper standard for folder naming yet. Franck reorganised a couple and seems to have a global idea of a structure, so i'll defer to him and follow his nomenclature